src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/ZeroFreeCoTaskMemUTF8Tests.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class ZeroFreeCoTaskMemUTF8Tests
+    {
+        [Fact]
+        public void ZeroFreeCoTaskMemUTF8_ValidPointer_Success()
+        {
+            IntPtr ptr = Marshal.StringToCoTaskMemUTF8("hello");
+            Marshal.ZeroFreeCoTaskMemUTF8(ptr);
```

Comment:
Please ZeroFree* API for memory allocated by Marshal.SecureStringTo* and use CoTaskMemoryFree* API for CoTaskMem

Reply:
My point is that there are no SecureString overloads for UTF8. This was an issue discussed in my previous comment. The only way we can get any test coverage for this method then to use the String API. I’ve modified the other APIs as you requested.

Reply:
Thanks.